src/core/send.c

```diff
@@ -1040,6 +1042,38 @@ QuicSendPathChallenges(
             continue;
         }
 
+#ifdef QUIC_USE_RAW_DATAPATH
+        //
+        // Make sure the route is resolved before sending the path challenge.
```

Comment:
Any ideas on how? Could we make a test to explicitly trigger this?

Reply:
You meant how a route is not resolved before sending the path challenge? Isn't this already covered by existing address rebind test cases? When we create a new path upon seeing a different source address from the peer, we create a path with an unresolved route.

Reply:
Ok.